tests/system/Database/Live/LikeTest.php

```diff
@@ -79,6 +79,12 @@ public function testLikeCaseInsensitive(): void
     #[DataProvider('provideMultibyteCharacters')]
     public function testLikeCaseInsensitiveWithMultibyteCharacter(string $match, string $result): void
     {
+        if ($this->db->DBDriver === 'SQLSRV') {
+            $this->markTestSkipped(
+                'Currently Builder class does not fully support Unicode strings in SQLSRV.'
```

Comment:
I was going to post the details as a bug report, but I gave it another try after your comment.

You are right. When I was checking this, I set collation only for the `value` field. Apparently, we have to set it for the entire database to work properly. Because of this, I was only able to add Unicode string using `N` prefix.

Thank you!

Reply:
Thanks for the explanation.